phoebe/frontend/bundle.py

```diff
@@ -3781,6 +3784,8 @@ def run_checks_compute(self, compute=None, solver=None, solution=None, figure=No
 
             # NOTE: atms are not attached to datasets, but per-compute and per-component
             for atmparam in self.filter(qualifier='atm', kind='phoebe', compute=computes, **_skip_filter_checks).to_list() + self.filter(qualifier='ld_coeffs_source').to_list():
+                if '_default' in atmparam.twig:
+                    continue
```

Comment:
```suggestion
            for atmparam in self.filter(qualifier='atm', kind='phoebe', compute=computes, check_visible=True, check_default=False).to_list() + self.filter(qualifier='ld_coeffs_source').to_list():
```

Reply:
I adopted this suggestion but I find it *very* obfuscated (not in a million years would I recognize that `check_visible=True` skips the '_default' pset). So, as a compromise, I added a comment in the code that explains this.